config.dsc

```diff
@@ -121,10 +121,7 @@ config({
                 { id: "Newtonsoft.Json", version: "11.0.2" },
                 { id: "Newtonsoft.Json", version: "10.0.3", alias: "Newtonsoft.Json.v10" },
                 { id: "Newtonsoft.Json.Bson", version: "1.0.1" },
-                { id: "System.Data.SQLite", version: "1.0.109.2" },

```

Comment:
how come that all of a sudden we don't need this?

Reply:
I guess it’s because you disabled DB init for aria 😀

Reply:
only on Windows; on macOS it should be just the same as before

Reply:
It is not used at all, there were only references to the .Core pacakge

---
In reply to: [310282058](https://github.com/microsoft/BuildXL/pull/696#discussion_r310282058) [](ancestors = 310282058)

Reply:
Note used, see other comment

---
In reply to: [310281164](https://github.com/microsoft/BuildXL/pull/696#discussion_r310281164) [](ancestors = 310281164)